### PR TITLE
[UT](test) add BE UT test case

### PR DESCRIPTION
**Pull request by zhangstar333**
## Proposed changes

Issue Number: close #xxx

### CODE REVIEW

**Comment by zhangstar333** [comment]
run buildall

**Comment by zhangstar333** [comment]
run buildall

File: be/test/vec/core/accurate_comparison_test.cpp [hunk 1]
```diff
@@ -0,0 +1,74 @@
+
```
Comment (HappenLee) [inline be/test/vec/core/accurate_comparison_test.cpp:1]: remove the empty line

**Comment by zhangstar333** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by Mryange — APPROVED** [review]

**Pull request merged**